py/desitarget/cuts.py

```diff
@@ -749,13 +762,14 @@ def isSTD(gflux=None, rflux=None, zflux=None, primary=None,
             https://gea.esac.esa.int/archive/documentation/GDR2/Gaia_archive/chap_datamodel/sec_dm_main_tables/ssec_dm_gaia_source.html
         - Current version (08/01/18) is version 121 on the wiki:
             https://desi.lbl.gov/trac/wiki/TargetSelectionWG/TargetSelection?version=121#STD
+
     """
     if primary is None:
         primary = np.ones_like(gflux, dtype='?')
     std = primary.copy()
 
     #ADM apply the Legacy Surveys (optical) magnitude and color cuts.
-    std &= isSTD_colors(primary=primary, zflux=zflux, rflux=rflux, gflux=gflux)
+    std &= isSTD_colors(primary=primary, zflux=zflux, rflux=rflux, gflux=gflux, south=south)
```

Comment:
Ultimately, we'll need bits like `STD_FAINT_NORTH`, `STD_BRIGHT_SOUTH` etc. to be set. For now, let's leave this. I'm not sure that the standard stars will need different optical selections as they're point sources and they're quite far from the flux limits of the imaging. If the MWS team decides that separate optical cuts *are* necessary, we'll update the bits then.